### ISSUE TITLE
update docs for mean/sum option

### ISSUE THREAD

**Issue by briley-lewis** [UCSB-Exoplanet-Polarimetry-Lab/GRaTeR-JAX#212]
Mihir switched this to mean because of "issues with the gradient descent fitting due to very large numbers"

Briley switched it to sum because that made it easier for the MCMC to explore the posterior without getting stuck in a fairly flat posterior space (so she thinks!)

which of these is the "correct" thing to do? I know statistically the log likelihood is typically the _sum_ of individual log likelihoods, but from some googling it looks like ML often uses mean. 

do we maybe want to have an option for users to specify which they want to use with a kwarg? maybe we do mean for gradient descent, sum for MCMC? @maxwellmb would love your thoughts

**Comment by briley-lewis** [comment]
relevant to PR #208

**Comment by briley-lewis** [comment]
updated so that mean is used for gradients, sum is used for mcmc, but user can also specify if they so desire (and things are still able to be jit compiled) -- this is in the same PR mentioned above. @mihir-r-kondapalli is also going to do some investigating as to why one converges better than the other in different cases

**Comment by briley-lewis** [comment]
@mihir-r-kondapalli has an explanation, needs to add this to docs -- repurposing this issue to that

from slack: "Looked into it, using the mean is better because it keeps the log-likelihood and its gradient on a consistent scale, regardless of image size. With the sum, both the objective function and gradient grow with the number of pixels, so a learning rate, gtol, or ftol that works for one image may fail for another. That makes optimization much less stable and forces the user to retune hyperparameters whenever the image size or dataset size changes.
With the mean, the scale stays normalized, so the same learning rate and tolerances are much more likely to work across different problems. The sum works better for MCMC since the sums are more precise than the means."
